packages/light-client/test/mocks/LightclientServerApiMock.ts

```diff
@@ -17,38 +18,46 @@ export class ProofServerApiMock implements routes.proof.Api {
   }
 }
 
+type VersionedLightClientUpdate = {
```

Comment:
stray type still here

Reply:
It was intentionally added to prevent having to repeat it a couple of times in the test. You will notice this is in a test file and also not exported. So it is a convenience. Feel free to remove if you think it is a blocker to merging. 🙏